classes/ft-ps-client.php

```diff
@@ -120,7 +120,7 @@ function init_premium_support() {
 
 							// We have a premium support license for this domain / product combination. Set the transient and property
 							set_transient( $status_key, $response->support_status, 60 * 60 * 24 );
-							//set_transient( $status_key, $response->support_status, 60 );	
+							//set_transient( $status_key, $response->support_status, 60 );
```

Comment:
__[code_sniffer]__
* Spaces must be used to indent lines; tabs are not allowed

<!-- Comment by [SideCI](https://sideci.com) -->